shop4All-backend/src/main/java/com/example/shop4All_backend/Controller/UserController.java

```diff
@@ -19,17 +19,32 @@ public class UserController {
     @Autowired
     private UserService userService;
 
+
+
     //initialize a demo admin
     @PostConstruct
     public void initRolesAndUsers(){
         userService.initRolesandUser();
     }
```

Comment:
Delete line 22 and 23 since we don't need so much space in our code.

Be careful at indentation.

Structure:

public void initRolesAndUsers() {

This way you ensure readability of the code and avoid branch conflicts caused by bad indentation of the code. Try to indent all code using IDE plugins or functionalities.